make.ps1

```diff
@@ -4,6 +4,11 @@ Param(
     [String] $target = "package"
 )
 
+# TODO: Remove when MSI code signing certificate is available
+# echo "Not packaging Windows components until code signing certificate is available"

```

Comment:
The # is a comment character, so this wouldn't do anything.

Reply:
Thanks!  Fixed in 984bdc5bba39cac9aae03cf2486914d32e6b1f03